calico-cloud_versioned_docs/version-3.16/image-assurance/understanding-scan-results.mdx

```diff
@@ -2,111 +2,97 @@
 
-**Enable Runtime View**
+**What is Download Scanner?**
 
-You can configure your scan settings to directly monitor pods that are running in your Kubernetes clusters.
-When enabled, Runtime View checks image information from the active container images in all of your connected clusters.
-These images are correlated with the images you have recorded in the Image Assurance system.
+This link goes to steps to integrate the Image Assurance scanner as part of your build pipeline. If you want to scan images only in your Kubernetes cluster, see [Enable automatic scanning inside a cluster](./scanners/cluster-scanner).
 
-To enable Runtime View, select the **Enable Runtime View** checkbox in the **Scan Results** tab.
-You can clear the checkbox to disable Runtime View.
-After disabling Runtime View, some runtime information for existing images will remain in the system.
+### Scan settings panel
 
-:::note
+**What is the Enable Runtime View checkbox?**
 
-If your cluster does not use the default containerd socket path (`/run/containerd/containerd.sock`), then you must change the path to allow the Runtime View service to collect image information.
+Runtime view allows you to view vulnerable images that are running in your cluster. You can then remediate the vulnerabilities using the CVSS scores as a guideline to triage CVEs.
 
-To update the CRI socket path for a cluster, run the following command:
+If you disable Runtime View, some runtime information for existing images will remain in the system.
 
-```bash
-kubectl patch imageassurance default --type='merge' -p '{"spec":{"criSocketPath":"<path-to-cri-socket>"}}'
-```
-See [the installation reference](../reference/installation/api.mdx#image-assurance.operator.tigera.io/v1.ImageAssuranceSpec) for more details on the ImageAssurance CR.
+**What are the `API URL` and `API Token` fields used for?**
 
-:::
+They are used to authenticate the connection to the Image Assurance scanner if you have integrated Image Assurance scanner in your build pipeline.
 
-### Exclude specific vulnerabilities
+### Exclude vulnerabilities from scan results
 
 In some cases, you may want to exclude specific vulnerabilities from your scan results. For example, you may determine that a warning is a false positive because of the way you are using an affected package. By creating a vulnerability exception, you can configure Image Assurance to ignore a CVE for a specific image, all images from the same repo, or for all images.
 
+1. Go to **Image Assurance**, **All Scanned Images**. 
+2. Select an image from the list.
+3. On the details page for that image, find the CVE, and click **Actions**, **Add Exception**.
+4. From the **Add Exception** dialog, select a scope, add a description, and click **Save**. The new vulnerability exception is immediately applied to your scan results.
+5. To view all vulnerability exceptions, go to: **Image Assurance**, **Vulnerability Exceptions**.
```

Comment:
These steps will need to change slightly, because with package-centric results the sequence is now: 
- ...
- Select image from list 
- Click on one of the listed packages in the side panel
- Pick one of the CVEs and click Actions, Add Exception 
- ...

Reply:
Fixed.